selfupdate/detect.go

```diff
@@ -127,7 +138,7 @@ func (up *Updater) DetectLatest(slug string) (*Release, bool, error) {
 	log.Println("Successfully fetched the latest release. tag:", rel.GetTagName(), ", name:", rel.GetName(), ", URL:", rel.GetURL(), ", Asset:", url)
 
 	publishedAt := rel.GetPublishedAt().Time
-	release := &Release{
+	release = &Release{
```

Comment:
Please continue to use `:=` here.

Reply:
It will not compile, `release` is already defined.